dotnet/src/Extensions/Extensions.UnitTests/PromptTemplates/Handlebars/Helpers/KernelSystemHelpersTests.cs

```diff
@@ -57,12 +57,62 @@ public async Task ItRendersTemplateWithJsonHelperAsync()
 
         // Act
         var result = await this.RenderPromptTemplateAsync(template, arguments);
-        result = result.Replace("&quot;", "\"", StringComparison.CurrentCultureIgnoreCase);
 
         // Assert
         Assert.Equal("{\"name\":\"Alice\",\"age\":25}", result);
     }
 
+    [Fact]
+    public async Task GetHelperWithTwoArgumentsReturnsPropertyValueAsync()
+    {
+        // Arrange
+        var template = @"{{get ""person"" ""name""}}";
+        var arguments = new KernelArguments
+            {
+                { "person", new { name = "Alice", age = 25 } }
+            };
+
+        // Act
+        var result = await this.RenderPromptTemplateAsync(template, arguments);
+
+        // Assert
+        Assert.Equal("Alice", result);
+    }
+
+    [Fact]
+    public async Task GetHelperWithHashParameterReturnsObjectAsync()
+    {
+        // Arrange
+        var template = @"{{get name=""person""}}";
+        var arguments = new KernelArguments
+            {
+                { "person", new { name = "Alice", age = 25 } }
+            };
+
+        // Act
+        var result = await this.RenderPromptTemplateAsync(template, arguments);
+
+        // Assert  
+        Assert.Equal("{ name = Alice, age = 25 }", result);
+    }
+
+    [Fact]
+    public async Task GetHelperWithNestedObjectReturnsNestedObjectAsync()
+    {
+        // Arrange  
+        var template = @"{{get ""person.Address""}}";
```

Comment:
Does "person.Address.City" also work?

Reply:
No, it just limits at the first level rn